common/men_at_arms_types/temp_draft_doc.txt

```diff
@@ -387,20 +358,19 @@ mushan_rider = {
 	screen = 35
 	
 	terrain_bonus = {
-		plains = { damage = 20 toughness = 15 pursuit = 10 }
-		drylands = { damage = 20 toughness = 15 pursuit = 10 }
-		desert = { damage = 20 toughness = 15 pursuit = 10 }
-		hills = { damage = -5 }
-		jungle = { damage = 5 }
-		mountains = { damage = -10 }
-		desert_mountains = { damage = -10 }
-		wetlands = { damage = -10 }
+		plains = { damage = 20 toughness = 20 pursuit = 15 screen = 15 }
+		drylands = { damage = 20 toughness = 20 pursuit = 15 screen = 15 }
+		desert = { damage = 20 toughness = 20 pursuit = 15 screen = 15 }
+		hills = { damage = -20 }
+		mountains = { damage = -20 }
+		desert_mountains = { damage = -20 }
+		wetlands = { damage = -20 }
 	}
 
 	winter_bonus = {
-		mild_winter = { damage = -4 toughness = -4 pursuit = -4 screen = -4}
-		normal_winter = { damage = -5 toughness = -5 pursuit = -5 screen = -5 }
-		harsh_winter = { damage = -6 toughness = -6 pursuit = -6 screen = -6 }
+		mild_winter = { damage = -15 toughness = -15 pursuit = -15 screen = -15 }
+		normal_winter = { damage = -18 toughness = -18 pursuit = -18 screen = -18 }
+		harsh_winter = { damage = -20 toughness = -20 pursuit = -20 screen = -20 }
```

Comment:
Better then before, though I'd vote that at least harsh winters should be even worse for cavalry.